build-tests/api-extractor-scenarios/etc/defaultExportOfEntryPoint4/rollup.d.ts

```diff
@@ -1,3 +1,4 @@
+/** @public */
```

Comment:
Why'd this change?

Reply:
I think this is just a compiler difference